site/assets/scss/_component-examples.scss

```diff
@@ -176,9 +176,9 @@
     display: inline-block;
     width: 9rem; // Boosted mod
     font-weight: $font-weight-bold; // Boosted mod
-    color: $gray-800; // Boosted mod: instead of `var(--bs-secondary-color)`
-    background-color: $gray-400; // Boosted mod: instead of `var(--bs-tertiary-bg)`
-    border: var(--bs-border-width) solid $gray-500; // Boosted mod
+    color: var(--bs-secondary-color);
+    background-color: var(--bs-secondary-bg); // Boosted mod: instead of `var(--bs-tertiary-bg)`
+    border: var(--bs-border-width) solid var(--bs-border-color-translucent); // Boosted mod: instead of `var(--bs-border-color)`
```

Comment:
```suggestion
    border: var(--bs-border-width) solid var(--bs-border-color-translucent); // Boosted mod: instead of `var(--bs-border-width) solid var(--bs-border-color)`
```